cg/constants/pacbio.py

```diff
@@ -17,6 +17,15 @@ class PacBioDirsAndFiles:
     RUN_IS_VALID: str = "is_valid"
 
 
+class MetricsFileFields:
+    ATTRIBUTES: str = "attributes"
```

Comment:
Just a minor thing and not anything you need to change but is there a reason for not considering this a class instantiation and foregoing the type hint? To me it is just as clear that this is a string as a custom object instantiation.

Reply:
I think it helps in PyCharm when assigning this to any variable. I don't see any other benefit though